Bottom: de7a9c05f4b76ef4a7c5f1e8c77095f8758f01fe
Top:    925a2bc0bbb9fb588876d7f6e99a9731add65afd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-21 21:56:05 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
